Dockerfile

```diff
@@ -259,7 +274,7 @@ ENV LOCALSTACK_BUILD_DATE=${LOCALSTACK_BUILD_DATE}
 ENV LOCALSTACK_BUILD_GIT_HASH=${LOCALSTACK_BUILD_GIT_HASH}
 ENV LOCALSTACK_BUILD_VERSION=${LOCALSTACK_BUILD_VERSION}
 
-# clean up some libs (e.g., Maven should be no longer required after "make init" has completed)
+# clean up some libs (e.g., Maven should be no longer required after initial installation has completed)
 RUN rm -rf /usr/share/maven
```

Comment:
Just noticed that. Shouldn't we include this in the lpm install call? Removing it in a separate layer seems wasteful storage wise.